exercises/pig-latin/src/test/java/PigLatinTranslatorTest.java

```diff
@@ -53,6 +58,15 @@
                 // Xr is treated like a single vowel
                 {"xray", "xrayay"},
 
+                // 'Y' is treated like a consonant at the beginning of a word
+                {"yellow", "ellowyay"},
+
+                // 'Y' is treated like a vowel at the end of a consonant cluster
+                {"rhythm", "ythmrhay"},
```

Comment:
So this line is why the build failed. Below is what is showed on the Travis Tests.
PigLatinTranslatorTest > test[19: expected "rhythm" to translate to the pig latin phrase "ythmrhay"] FAILED
    org.junit.ComparisonFailure: expected:<[ythmrh]ay> but was:<[hythmr]ay>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at PigLatinTranslatorTest.test(PigLatinTranslatorTest.java:82)

Not sure why it failed though, because the the canonical-data lists exactly that as the test case. Maybe someone from @exercism can look into it more and find out why.

Reply:
As said above - just because the test is the correct test, doesn't mean the build will pass!

We also need the reference solution store in `.meta` to pass all of the tests as well. As we can see here, all of the tests pass apart from this one (probably because this 'y' is an edge case where it will be treated as a vowel after a consonant).

The current version of the reference solution gives us `hythmray` instead of `ythmrhay`!

Reply:
As said above - just because the test is the correct test, doesn't mean the build will pass!

We also need the reference solution store in `.meta` to pass all of the tests as well. As we can see here, all of the tests pass apart from this one (probably because this 'y' is an edge case where it will be treated as a vowel after a consonant).

The current version of the reference solution gives us `hythmray` instead of `ythmrhay`!

Reply:
Thanks @sjwarner-bp for the explanation, I was writing this comment at the same time you posted the one on the PR so I didn't see that one.

Reply:
Alright, thanks @sjwarner-bp and @c-thornton! I'll update it to "hythmray." How can I run the tests locally?

Reply:
@Kyle-P I remember @sjwarner-bp made a comment a while ago about it. I don't know who to link to it directly but it was in PR #1222. Basically you need to install gradle and run the tests locally. You can find some more information about how to set it up in the contributing guide found here.

https://github.com/exercism/java/blob/master/CONTRIBUTING.md#advanced-complete-local-setup

Reply:
Thansk @c-thornton! I'd rather keep things done online for personal reasons, but I definitely see how it's better to run tests locally. Would setting up TravisCI on my fork be a plausible option?